R/layer_population_scaling.R

```diff
@@ -0,0 +1,133 @@
+#' Revert population scaled prediction
+#'
+#' @param frosting a `frosting` postprocessor
+#' @param ... the column(s) in the `epi_df` to scale back.
+#' @param df a data frame that contains the population data used for scaling.
+#' @param by A character vector of variables to join by. Column names should be
+#' in lower case.
```

Comment:
Yah, that's true. I'll change this description. The upper/lowercase consistency is adjusted for in the functions so that the `join` can be performed smoothly. I have a unit test that checks this functionality.